gtest/Cargo.toml

```diff
@@ -8,6 +8,7 @@ license.workspace = true
 
 [dependencies]
 gear-core.workspace = true
+gear-common = { workspace = true, features = ["std"] }
```

Comment:
It might be `gtest` is not so lightweight now. Could you please compare compilation time of the crate?